### PR TITLE
update paul revere gem to latest

### PR DESCRIPTION
**Pull request by Ch4s3**

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
Convenience links for reviewer: [paul_revere v1.4...v2.0.0](https://github.com/thoughtbot/paul_revere/compare/v1.4...v2.0.0) [CHANGELOG](https://github.com/thoughtbot/paul_revere/blob/master/NEWS.md).

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -97,7 +97,7 @@
       </div>
     <% end %>
 
-    <%= render :partial => "announcements/announcement_for_all" %>
+    <%= render :partial => "announcements/public_announcement" %>
```
Comment (JuanitoFatas) [inline app/views/layouts/application.html.erb]: Could you update to 1.9+ hash syntax? :blush:

Comment (Ch4s3) [inline app/views/layouts/application.html.erb]: Sure 😅

**Comment by Ch4s3** [comment]
I fixed the hash syntax. Is it worth putting in a separate PR later that updates the hash syntax everywhere?

File: vendor/assets/javascripts/announcements.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+function hideAnnouncement(announcement_key) {
```
Comment (arthurnn) [inline vendor/assets/javascripts/announcements.js]: why we had to add this file ?

Comment (Ch4s3) [inline vendor/assets/javascripts/announcements.js]: The docs implied that the js file was required. After testing this, it seems that the js file and require are unnecessary.

**Comment by qrush** [comment]
:ok_hand:

**Comment by qrush** [comment]
Let's get this rebased, toss out the `vendor` js addition and we can merge it in.

**Comment by Ch4s3** [comment]
@qrush should be good now

**Comment by Ch4s3** [comment]
ping

**Comment by dwradcliffe** [comment]
:+1:

**Comment by dwradcliffe** [comment]
@Ch4s3 Looks like we still need to include the js from the gem. We probably just need to add a line in `app/assets/javascripts/application.js`. Can you take a look?

**Pull request merged**